### ISSUE TITLE
Bugs in Prod

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/network#393]
1. NEWS PAGE - "This encryption tool can make you smile" news links to a German link - title should also be in German to set the proper expectation: https://www.wired.de/collection/tech/dieses-verschluesselungs-tool-laesst-eure-texte-laecheln

2. PEOPLE PAGE  - The Featured Cards have icons but the links don't work in DESKTOP - work on MOBILE (also there is no "Read Interview" link - is this still blocked by mozilla/network-api#168)? - **ticket #394**

3. SUPPORT PAGE - link "See grant project examples (NYC/Chicago)" links to Hive Chicago - so maybe we should remove NYC from the link? - **ticket #395** 

4. UPCOMING PAGE - "Mozfest" is marked as "November 2017" when actually is "October 2017" - **ticket #397**

**Comment by taisdesouzalessa** [comment]
@hannahkane not sure which ones are priority to fix here. Could you help me to sort those out?

**Comment by hannahkane** [comment]
Thanks, @taisdesouzalessa.

I just addressed 1 in the CMS, but now I'm wondering if I followed the right protocol. @alanmoo - can you please advise? Was it ok for me to go into prod admin and make that change directly, or should I have done that in staging admin first?

2 definitely seems important. @taisdesouzalessa - can you pull that out into a separate ticket?

3 - good catch! @taisdesouzalessa - you should definitely file a PR for that one!

4 - according to #127, the intent was to replace the single "SEE GRANT PROJECT EXAMPLES (NYC/CHICAGO)" with three separate lines. [See funded project examples from Hive NYC](http://hivenyc.org/portfolio/), [See funded project examples from Hive Chicago](http://hivechicago.org/portfolio/#/*), and [See funded project examples from Hive Toronto](http://hivetoronto.org/portfolio/). Is that something you could file a PR for?

**Comment by taisdesouzalessa** [comment]
Status:
- tickets and PRs filed for 3 (#395) and 4(#397). 
- separate ticket filed for 2: #394

**Comment by alanmoo** [comment]
> I just addressed 1 in the CMS, but now I'm wondering if I followed the right protocol. @alanmoo - can you please advise? Was it ok for me to go into prod admin and make that change directly, or should I have done that in staging admin first?

That's the right approach- it'll be auto deployed to staging so you can see the effect of the changes, and then we can promote staging to production when it's ready.

**Comment by hannahkane** [comment]
ok, thanks, @alanmoo. On staging the headline looks....German. So, good to go whenever the next push to prod happens.

**Comment by hannahkane** [comment]
Closing this ticket, since everything is addressed in one way or another.
